createdb.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python
 
+from __future__ import print_function
```

Comment:
On Mon, Dec 21, 2015 at 03:02:17PM -0800, Craig Rodrigues wrote:
>    In createdb.py:
> 
>  > @@ -1,5 +1,7 @@
>  >  #!/usr/bin/python
>  > 
>  > +from __future__ import print_function
> 
>    Just for completeness, I'd rather keep that in there, because on Python2,
>    print() has different semantics unless you use future.

I was not aware of this, could you expand?

Thanks